internal/regopolicyinterpreter/regopolicyinterpreter.go

```diff
@@ -301,18 +315,22 @@ func (r *RegoPolicyInterpreter) EnableLogging(path string, level LogLevel) error
 	r.resultsLogger = log.New(file, "RESULT: ", log.Ldate|log.Ltime)
 	r.metadataLogger = log.New(file, "METADATA: ", log.Ldate|log.Ltime)
 	r.logInfo("Logging Enabled at level %d", level)
+
+	r.compiledModules = nil
 	return nil
 }
 
 // SetLogLevel sets the logging level. To actually produce a log, however, EnableLogging
 // must be called first.
 func (r *RegoPolicyInterpreter) SetLogLevel(level LogLevel) {
 	r.logLevel = level
+	r.compiledModules = nil
 }
 
 // DisableLogging disables logging and closes the underlying log file.
 func (r *RegoPolicyInterpreter) DisableLogging() error {
 	r.logLevel = LogNone
+	r.compiledModules = nil
```

Comment:
Same as `EnableLogging`, should dataAndModulesMutex be held when resetting `compiledModules`?

Reply:
Done.